AUTHORS

```diff
@@ -1,14 +1,15 @@
 The following people have contributed to the project (in alphabetical order):
 
-- Andreas Salzburger, CERN
-- Bastian Schlag, CERN, JGU Mainz
-- Christian Gumpert
+- Xiaocong Ai, DESY
 - Corentin Allaire, CERN
-- Fabian Klimpel, Max-Planck-Institut für Physik
+- Noemi Calace, CERN
+- Paul Gessinger, JGU Mainz
 - Hadrien Grasland, Centre National de la Recherche Scientifique
```

Comment:
While we're at it, let's also do...

```suggestion
- Hadrien Grasland, CNRS
```

...since we're using acronyms for all the others ;)